apps/src/code-studio/initApp/project.js

```diff
@@ -84,6 +84,7 @@ let newSourceVersionInterval = 15 * 60 * 1000; // 15 minutes
 var currentAbuseScore = 0;
 var sharingDisabled = false;
 var currentHasPrivacyProfanityViolation = false;
+var currentShareFailure = false;
```

Comment:
nit: i think the default value here should be a string (`var currentShareFailure = '';`) since the actual content from `/share-failure` will be a string